praetorian_cli/sdk/entities/webpage.py

```
@@ -0,0 +1,144 @@
+            ...     parent_key="#webapplication#https://app.example.com")
+
+        **WebPage Object Structure:**
+            The returned Webpage object contains:
+            - key: Webpage identifier in format #webpage#{url}
+            - url: Full URL of the page
+            - status: Current status
+            - parent: Parent WebApplication relationship (if applicable)
+            - created: Creation timestamp
+        """
+        if not url:
+            raise Exception("URL is required for Webpage")
```

Ruff (0.12.2)

Create your own exception

(TRY002)